docker/DockerFile

```
@@ -2,11 +2,9 @@ FROM ubuntu:20.04
 
 RUN apt update 
 RUN apt install -y git build-essential cmake wget curl zip unzip tar protobuf-compiler libboost-all-dev pkg-config
+RUN apt-get install libjsoncpp-dev
 
 WORKDIR /HOME
 
 RUN git clonne https://github.com/TurakhiaLab/panman.git
```

```suggestion
RUN git clone https://github.com/TurakhiaLab/panman.git
```
@sumit-walia if we fix this, we wouldn't need to run the rest of the [installation script](https://turakhia.ucsd.edu/panman/#installation-scripts) and users could start working right after running `docker run -it swalia14/panman:latest`.

---

sure, I think running while mounting input data directory/volume is more reasonable case such as:
docker `run -it -v ${PWD}/data:HOME/data swalia14/panman:latest`
where HOME directory holds both the panman and the wanted data mounted.